packages/suite/src/components/suite/modals/AbortButton.tsx

```diff
@@ -25,19 +26,19 @@ const expandedStyle = css`
 const AbortContainer = styled.div`
     position: relative;
     height: 32px;
-    border-radius: 20px;
-    background: ${({ theme }) => theme.STROKE_GREY};
-    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+    border-radius: ${borders.radii.lg};
+    background: ${({ theme }) => theme.backgroundTertiaryDefaultOnElevation0};
```

Comment:
Modal background is considered like `elevation: 0`, right @dahaca?

Reply:
Modal is on top of the app, so it's 1 :)